main.cpp

```diff
@@ -104,33 +104,33 @@ app.crashToDesktop();
 //  action 1:   releases compressed air via the pistol
 airComp.fire();
 //  action 2:   adjusts output pressure
-airComp.outputPressure();
+airComp.adjustOutputPressure();
 //  action 3:   adjusts maximum pressure inside the main tank
-airComp.tankPressure(); 
+airComp.adjustTankPressure(); 
 //  8)
 //  Noun:       microphone
 //  action 1:   records audio
-microphone.record();
+microphone.recordAudio();
 //  action 2:   produces noise
-microphone.addNoise();
-//  action 3:   automatically adjusts preamp gain
-microphone.autoGain();
+microphone.produceNoise();
+//  action 3:   adjusts preamp gain
+microphone.adjustPreampGain();
 //  9)
 //  Noun:       eq
 //  action 1:   adjust band frequency
-eq.band1Freq();
+eq.adjustBand1Freq();
 //  action 2:   adjust band q
-eq.band1Q();
+eq.adjustBand1Q();
 //  action 3:   adjust band gain
-eq.band1Gain ();
+eq.adjustBand1Gain ();
 //  10)
 //  Noun:       reverb
 //  action 1:   adjust size
-reverb.size();
+reverb.adjustSize();
 //  action 2:   adjust rt60
-reverb.rt60();
+reverb.adjustRt60();
 //  action 3:   adjust predelay
- reverb.predelay();
+ reverb.adjustPredelay();
 

```

Comment:
Your code should line up with the left edge of the text editor
It should be indented when it is inside a matched pair of `{ }`
- [ ] revise